src/ForkCMS/ImagineController.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace ForkCMS;
+
+use Imagine\Exception\RuntimeException;
+use Liip\ImagineBundle\Exception\Binary\Loader\NotLoadableException;
+use Liip\ImagineBundle\Exception\Imagine\Filter\NonExistingFilterException;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\Exception\BadRequestHttpException;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+
+class ImagineController extends \Liip\ImagineBundle\Controller\ImagineController
+{
+    /**
+     * This action applies a given filter to a given image, optionally saves the image and outputs it to the browser at the same time.
+     *
+     * @param Request $request
+     * @param string  $path
+     * @param string  $filter
+     *
+     * @throws \RuntimeException
+     * @throws BadRequestHttpException
+     *
+     * @return RedirectResponse
+     */
+    public function filterAction(Request $request, $path, $filter)
```

Comment:
Would it not be possible to check if it is an svg, then handle it here yourself but if it isn't just to kick it up to the parent?